src/pages/spawn/spawnVolume/spawnVolumeTable/spawnVolumeTableActions/editButton/editVolumeModal/reducer.ts

```diff
@@ -13,7 +13,7 @@ export const getInitialState = (volume: MyVolume) => ({
   expiration: new Date(volume.expiration),
   volumeId: volume.id,
   noExpiration: volume.noExpiration,
-  name: volume.displayName,
+  name: volume.uiDisplayName,
```

Comment:
This will result in a volume always having a display name since this field falls back to an id.